WcaOnRails/spec/controllers/api_users_controller_spec.rb

```diff
@@ -78,7 +78,6 @@
       expect(response.status).to eq 200
       json = JSON.parse(response.body)
       expect(json["user"]).to eq competed_user.as_json
-      expect(json.key?("rankings")).to eq true
```

Comment:
This test case should not disappear. We might have external API users that rely on it.